app/Http/Controllers/ReactionController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Like;
+use App\Models\Rate;
+use Auth;
+use Illuminate\Http\Request;
+
+class ReactionController extends Controller
+{
+    public function react($bookId)
+    {
+        $user = Auth::user();
+        $likes = Like::where('user_id', $user->id)->where('book_id', $bookId)->first();
+        $book_like = Like::where('book_id', $bookId)->get();
```

Comment:
```suggestion
        $bookLike = Like::where('book_id', $bookId)->get();
```
mà book like nghĩa là gì e?